### ISSUE TITLE
Deployed app shows iron:router welcome screen, local app works fine

### ISSUE THREAD

**Issue by dandv** [kahmali/meteor-restivus#43]
Reproduction repo: https://github.com/dandv/webix-restivus
Demo: http://webix-restivus.meteor.com/

The bug summary is the naive observation. Looking at the console shows a Reference error "Route is not defined" at  nimble-restivus.js line 124:

    route = new Route(this, path, options, methods);

When deployed with `--debug`, the deployed app also continues to run, and shows the grid, just as the locally run app does. BTW, should `--debug` cause this difference? CC @stubailo.

This is ultimately caused by [Restivus.addCollection being called on the client](https://github.com/dandv/webix-restivus/blob/master/lib/collections.js#L5) via `lib`.

**Comment by stubailo** [comment]
Should be fixed by #41

**Comment by kahmali** [comment]
Absolutely, @stubailo. I'm reviewing #41 now and it looks good so far (I'll post all this and more on the actual pull request later today). I just have an exam I need to cram for until Wednesday, so I may not be able to finish reviewing  it until then. Sorry for the hold up on this! I look forward to getting rid of all these `iron:router` issues once and for all.

**Comment by stubailo** [comment]
No problem, @kahmali! Take as much time as you need.

**Comment by kahmali** [comment]
This is resolved by #46. Thanks so much for reporting it!

**Comment by renewsoqv** [comment]
Im receiving the same erro on the file restivus.coffee

**Comment by kahmali** [comment]
@renewsoqv There's no reason to configure or define routes on the client. Just do everything on the server and you should be fine. I'll be sure to only export Restivus on the server and make that very clear in the next update. 

If you're not using Iron Router on the client, then you'll have to wait until v0.7.0 is published (or clone an early build from the 0.7.0 branch).

**Comment by renewsoqv** [comment]
@kahmali My config is as:

```
Restivus.configure
  useAuth: true
  prettyJson: true
  useClientRouter: false
  onLoggedIn: -> console.log "#{@user.email} (#{@userId}) logged in on api"
  onLoggedOut: -> console.log "#{@user.email} (#{@userId}) logged out on api"
```

On the common folder, i used it there following the instructions on docs. Should i move it to a server folder?

**Comment by kahmali** [comment]
@renewsoqv See #65

**Comment by renewsoqv** [comment]
@kahmali Thanks, placing the config on the server folder solved my issue, plus the reading of #65 really clarify everything.

**Comment by kahmali** [comment]
Sorry for leaving the docs out of sync once that bug was discovered! I should have removed that information. I've created a new issue and I'll fix it in the next update (0.6.6).
